src/monitor-view-events.js

```diff
@@ -9,25 +9,32 @@ import { triggerMethodOn, triggerMethodMany } from './trigger-method';
 // whenever a rendered view is attached or an attached view is rendered.
 function MonitorViewEvents(view) {
   if (view._areViewEventsMonitored) { return; }
+
+  if (!view._getImmediateChildren) {
+    return;
+  }
```

Comment:
How does a view get to this point if it doesn't have this function?  This doesn't feel right to me.  I would expect that MonitorViewEvents would only be applied to views that have it.